bindings/java/README.md

```diff
@@ -41,3 +41,9 @@ try (TranslationModel model = Translatador.builder().model(modelPath).vocab(voca
 ```
 
 You can find pre-built open-source models optimized for the CPU in the [firefox-translation-models](https://github.com/mozilla/firefox-translations-models) repository.
+
+Built on [whatlang-rs](https://github.com/greyblake/whatlang-rs), Translatador can detect 67 different languages:
```

Comment:
I think it might be nice to link the supported languages directly as well (https://github.com/greyblake/whatlang-rs/blob/master/SUPPORTED_LANGUAGES.md), and it seems 69 languages are supported now instead of 67?